plugins/Monolog/Formatter/LineMessageFormatter.php

```diff
@@ -43,6 +48,21 @@ public function format(array $record)
 
         $message = trim($record['message']);
 
+        if (gettype($this->excludePatterns) === "array") {
```

Comment:
with the constructor mandated type of the param, this can just be
```suggestion
        if (!empty($this->excludePatterns)) {
```